elfi/methods/bo/gpflow_regression.py

```diff
@@ -0,0 +1,312 @@
+"""This module contains an interface for using the GPflow library in ELFI."""
+
```

Comment:
Hmm... perhaps the current `gpy_regression.py` could be refactored to `gp_regression.py` that includes also this?